doc/newsboat.asciidoc

```diff
@@ -648,6 +648,10 @@ In addition, Miniflux requires username and password for authentication:
 	miniflux-login "myusername"
 	miniflux-password "mypassword"
 
+Alternatively, an API Token can be specified:
```

Comment:
I think this paragraph should be further down, because the next two are related to passwords specifically.

Also, the paragraph about basic HTTP authentication should be updated to reflect that this auth is only used with passwords.

Reply:
Makes sense, done